plugin/validator_test.go

```diff
@@ -2378,3 +2378,63 @@ func getErrForField(e error, path string) string {
 	}
 	return ""
 }
+
+func TestValidator_Transformations(t *testing.T) {
```

Comment:
Can we assert the `v.Transform(plugin)` works when multiple transformations are defined on a schema?